Src/BootCamp.Chapter/Person.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Person
+    {
+        private readonly string[] _personAndBalanceArray;
+        private readonly float[] _balance;
+
+        public Person(string person)
+        {
+            _personAndBalanceArray = ArrayHandler.ConvertToArray(person);
+            _balance = ArrayHandler.ConvertToBalanceArray(person);
+        }
+
+        public string GetName()
+        {
+            return _personAndBalanceArray[0];
+        }
+
+        public float[] GetBalanceHistory()
+        {
+            return _balance;
+        }
+
+        public float GetCurrentBalance()
+        {
+            return _balance[^1];
+        }
+
+        public float GetHighestBalance()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetHighestBalance' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5070129)